### ISSUE TITLE
how to add gradient color in stacked bar chart

### ISSUE THREAD

**Issue by akshPairroxz** [Abhinandan-Kushwaha/react-native-gifted-charts#231]
hello , i would like to add a gradient color in stacked bar chart , anyone know how to implement it?

**Comment by Abhinandan-Kushwaha** [comment]
Hi @akshPairroxz 👋
Thanks for requesting this feature. It is now available in version `1.2.36`🎉

Below is a sample Stacked Bar chart.

![Screenshot 2022-06-20 at 19 20 52](https://user-images.githubusercontent.com/20596944/174616482-fe0b56a6-f8d7-44cd-b7e7-643658b379c5.png)

The code for the above chart is-
```js
const stackData = [
  {
    stacks: [
      {
        value: 10,
      },
      {
        value: 20,
        marginBottom: 2,
      },
    ],
    spacing: 15,
    label: 'Jan',
  },
  {
    stacks: [
      {value: 10},
      {
        value: 11,
        marginBottom: 2,
      },
      {
        value: 15,
        marginBottom: 2,
      },
    ],
    label: 'Mar',
  },
  {
    stacks: [{value: 14}, {value: 18, marginBottom: 2}],
    spacing: 15,
    label: 'Feb',
  },
  {
    stacks: [{value: 7}, {value: 11, marginBottom: 2}],
    label: 'Mar',
  },
];

return <BarChart stackData={stackData} />
```

Now, just add the prop `showGradient` to the `<BarChart>` component, and it results in-

![Screenshot 2022-06-20 at 19 23 50](https://user-images.githubusercontent.com/20596944/174617143-e944c478-68ec-4c2f-b961-92d80cd31db7.png)

It takes white as the gradient color by default. We can change it using the `gradientColor` prop.

The different bars can be given different values of `gradientColor` and `color`. Similarly, each section of a bar can be given different values of `gradientColor` and `color`. Here is an example-

![Screenshot 2022-06-20 at 19 28 41](https://user-images.githubusercontent.com/20596944/174618110-e99b4e99-3f18-468f-a628-057dc007d771.png)

The code for the above chart is-

```js
const stackData = [
  {
    stacks: [
      {
        value: 10,
      },
      {
        value: 20,
        marginBottom: 2,
      },
    ],
    spacing: 15,
    label: 'Jan',
  },
  {
    gradientColor: 'blue',
    stacks: [
      {value: 10},
      {
        value: 11,
        marginBottom: 2,
      },
      {
        value: 15,
        marginBottom: 2,
      },
    ],
    label: 'Mar',
  },
  {
    stacks: [{value: 14}, {value: 18, gradientColor:'green', marginBottom: 2}],
    spacing: 15,
    label: 'Feb',
  },
  {
    stacks: [{value: 7}, {value: 11, marginBottom: 2}],
    label: 'Mar',
  },
];

return <BarChart stackData={stackData} showGradient />
```

Also, instead of passing the `showGradient` prop directly to the `<BarChart>` component, we can pass it to individual bars or bar sections to apply gradient effect only to them.

I hope it helps.🤗

**Comment by akshPairroxz** [comment]
wow , that's great , thanks for the update.
